pages/api/v1/user-created-bounties/getAllBounties.ts

```
@@ -0,0 +1,119 @@
+
+export async function getUserCreatedBounties({ status, filterBy, page, network }: Args): Promise<IApiResponse<IUserCreatedBounty[] | MessageType>> {
+	try {
+		if (!network || !isValidNetwork(network)) throw apiErrorWithStatusCode(messages.INVALID_NETWORK, 400);
+
+		if (status && !Object.values(EUserCreatedBountiesStatuses).includes(status)) {
+			throw apiErrorWithStatusCode('Invalid Status Param', 400);
+		}
+		if (filterBy?.length && !!filterBy?.filter((tag: string) => typeof tag !== 'string')?.length) {
+			throw apiErrorWithStatusCode('Invalid Tags in Param', 400);
+		}
+		if (isNaN(page)) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)